tests/yaml_spec/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,2828 @@
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+- &😁 unicode anchor
+
+`;
+
+exports[`anchors-and-tags.yml 1`] = `
+ - &a !!str a
+ - !!int 2
+ - !!int &c 4
+ - &d d
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+- !!str &a a
+- !!int 2
+- !!int &c 4
+- &d d
+
+`;
+
+exports[`anchors-in-mapping.yml 1`] = `
+&a a: b
+c: &d d
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+? &a a
+: b
+c: &d d
+
+`;
+
+exports[`anchors-with-colon-in-name.yml 1`] = `
+&a: key: &a value
+foo:
+  *a:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+? &a: key
+: &a value
+foo: *a:
+
+`;
+
+exports[`backslashes-in-singlequotes.yml 1`] = `
+'foo: bar\\': baz'
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+'foo: bar\\': baz'
+
+`;
+
+exports[`bare-document-after-document-end-marker.yml 1`] = `
+---
+scalar1
+...
+key: value
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+scalar1
+...
+key: value
+
+`;
+
+exports[`blank-lines.yml 1`] = `
+foo: 1
+
+bar: 2
+    
+text: |
+  a
+    
+  b
+
+  c
+ 
+  d
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+foo: 1
+
+bar: 2
+
+text: |
+  a
+    
+  b
+
+  c
+
+  d
+
+`;
+
+exports[`block-mapping-with-missing-values.yml 1`] = `
+? a
+? b
+c:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+? a
+? b
+? c
+
+`;
+
+exports[`block-mapping-with-multiline-scalars.yml 1`] = `
+? a
+  true
+: null
+  d
+? e
+  42
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+a true: null d
+? e 42
+
+`;
+
+exports[`block-mappings-in-block-sequence.yml 1`] = `
+ - key: value
+   key2: value2
+ -
+   key3: value3
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+- key: value
+  key2: value2
+- key3: value3
+
+`;
+
+exports[`block-scalar-indicator-order.yml 1`] = `
+- |2-
+  explicit indent and chomp
+- |-2
+  chomp and explicit indent
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+- |2-
+  explicit indent and chomp
+- |2-
+  chomp and explicit indent
+
+`;
+
+exports[`block-scalar-keep.yml 1`] = `
+--- |+
+ ab
+ 
+  
+...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+|+
+  ab
+
+   
+
+`;
+
+exports[`block-scalar-strip.yml 1`] = `
+|-
+ ab
+ 
+ 
+...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+|-
+  ab
+
+`;
+
+exports[`block-scalar-with-spaces-only.yml 1`] = `
+empty block scalar: >
+ 
+  
+   
+ # comment
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+empty block scalar: >
+# comment
+
+`;
+
+exports[`block-sequence-in-block-mapping.yml 1`] = `
+key:
+ - item1
+ - item2
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+key:
```

Comment:
I usually don't indent these (I would write it like the input). Do you have opinions on this?

Reply:
I forgot to not indent it. 😂

Reply:
Cool. It will probably come up again as people probably have _very_ strong opinions both ways. 😆